packages/table/src/layers/_layers.scss

```diff
@@ -40,6 +40,10 @@ $region-selected-color: $pt-intent-primary !default;
   }
 }
 
+.bp-table-focus-region {
+  border: 2px solid $blue3;
```

Comment:
wonder if we've got variables for this. `$pt-intent-primary` at least? @llorca ?